src/style.css

```diff
@@ -1 +1,46 @@
 /* add styles here */
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  flex-direction: row;
```

Comment:
it's default value, so we can remove it

Reply:
You mean remove only padding? If i remove all this values picture with stars looks awful.

Reply:
If i remove padding, all tests are failure.